prisma/idx-template.json

```
@@ -1,9 +1,22 @@
 {
-    "name": "Prisma",
-    "description": "A simple postgres database starter",
-    "categories": [
-        "Databases"
-    ],
-    "icon": "https://www.gstatic.com/monospace/231115/logo_cpp.svg",
-    "publisher": "Google"
+  "name": "Prisma",
+  "description": "Get started with a Prisma Postgres or SQLite database using Prisma ORM.",
+  "categories": [
+    "Databases"
+  ],
+  "icon": "https://www.gstatic.com/monospace/231115/logo_cpp.svg",
```

Do you want it to be the C++ logo?

---

ouch, definitely not! thanks for catching that 🙏  here's a link to our logo in the [presskit on github](https://github.com/prisma/presskit/blob/main/Assets/Prisma-DarkSymbol.svg), does that work?

```suggestion
  "icon": "https://raw.githubusercontent.com/prisma/presskit/1c8937b2aefd6417515bcee30272a960df20a8d8/Assets/Prisma-DarkSymbol.svg",
```